tfhe-zk-pok/Cargo.toml

```diff
@@ -22,6 +22,7 @@ rayon = "1.8.0"
 sha3 = "0.10.8"
 serde = { version = "~1.0", features = ["derive"] }
 zeroize = "1.7.0"
+num-bigint = "0.4.5"
```

Comment:
in the end do you think it's better to pull that dependency ?

Reply:
yeah, doesn't seem to add any compile-time overhead and we can replace it if we ever need to in the future